include/boost/geometry/util/math.hpp

```diff
@@ -771,6 +772,106 @@ inline Result rounding_cast(T const& v)
     return detail::rounding_cast<Result, T>::apply(v);
 }
 
+/*!
+\brief Evaluate the sine and cosine function with the argument in degrees
+\note The results obey exactly the elementary properties of the trigonometric
+      functions, e.g., sin 9&deg; = cos 81&deg; = &minus; sin 123456789&deg;.
+      If x = &minus;0, then \e sinx = &minus;0; this is the only case where
+      &minus;0 is returned.
+*/
+template<typename T>
+inline void sin_cos_degrees(T const& x,
+                            T & sinx,
+                            T & cosx)
+{
+    // In order to minimize round-off errors, this function exactly reduces
+    // the argument to the range [-45, 45] before converting it to radians.
+    T remainder; int quotient;
+
+    remainder = std::fmod(x, T(360));
+    quotient = int(std::floor(remainder / 90 + T(0.5)));
+    remainder -= 90 * quotient;
+
+    // Convert to radians.
+    remainder *= d2r<T>();
+
+    T s = std::sin(remainder), c = std::cos(remainder);
```

Comment:
Before you used math functions without `std::`. If you specify `std::` then the compiler will not use functions from the namespace where a user-defined numeric type is defined. So please remove `std::`. If you like you can bring `std::` math functions into scope:

    using std::floor;
    using std::sin;
    using std::cos;

but they are not used like that in the whole library so I'd say this is not needed.